webapp/src/components/incoming_calls/call_incoming_condensed.tsx

```diff
@@ -80,13 +89,15 @@ export const CallIncomingCondensed = ({call, onWidget = false, joinButtonBorder
                     }
                     trigger={showTooltip ? ['hover', 'focus'] : []}
                 >
-                    <Message ref={messageRef}>
+                    <Message
+                        ref={messageRef}
+                    >
```

Comment:
Automatic formatting?

Reply:
During the investigation (which took a while), I temporarily set `data-testid` for this element, which I later removed since the Playwright locator wasn't the issue. Formatting is likely a result of that.